### ISSUE TITLE
Support for debugger extension in Pachyderm Notebooks

### ISSUE THREAD

**Issue by RaananHadar** [pachyderm/pachyderm#6899]
**What is the goal / desired outcome?**
Use some basic extensions in Pachyderm Notebooks such as the debugger extension.

**If there is a way to accomplish this today via workaround, what does that require?**
My understanding is that the current beta is using an old version of JupyterLab which prevents some extensions from installing successfully. So I guess it is possible in the on prem version to upgrade the helm chart to use the latest version, but there is not good solution for Pachyderm Hub.

**(Optional) What is your proposal for a feature to solve this?**
Upgrade JupyterLab to a more recent server might solve this.

**Environment?**:
Pachyderm Hub

**Comment by RaananHadar** [comment]
Yes, i've tried just now @chainlink. It is my experience that the current version of Hub at the time of writing does not come with a debugger preinstalled and trying to install the debugging extensions fail.

**Comment by chainlink** [comment]
@JimmyWhitaker mind taking a look here?
